.repo/sample-list-generator/src/index.ts

```diff
@@ -0,0 +1,15 @@
+import path from 'path';
+import fs from 'fs/promises';
+import { getAllSamples } from './libs/sample-collector';
+
+(async () => {
```

Comment:
Similar to the other file can we remove this and just have a `start()` function or similar? I'm not a huge fan of these immediately-invoked functions.